File: Desafios_Javascript/Solução_de_Problemas_Essenciais/desafio_03.js
/* Desafio 3 Ultrapassando V */

let r = parseInt(gets());
let v = 0;

while(v <= r){
  v = parseInt(gets());
}

let sum = 0;
let counter = 0;

for(sum = 0; sum <= v; r++, counter ++ ){
  sum += r;
}

console.log(counter);

File: Desafios_C_Sharp/Solução_de_Problemas_Básicos_ em_C_Sharp/desafio_03.cs
/* Desafio 3 - Validação de Nota */

using System;

class desafio_03 {
    static void Main() {
    
    float sum = 0;
    int counter = 0;
    bool end = false;
    
    while(!end)
    {
      float input = float.Parse(Console.ReadLine());
      
      if(input >= 0 && input <=10)
      {
        sum += input;
        counter++;
      }
      else
      {
        Console.WriteLine("nota invalida");
      }
      if(counter == 2)
      {
        float media = (sum/2);
        
        Console.WriteLine("media = " + media.ToString("N2"));
        
        end = true;
      }
    }
  }
}